src/resolve/test-dns-synthesize.c

```diff
@@ -0,0 +1,240 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "resolved-dns-answer.h"
+#include "resolved-dns-question.h"
+#include "resolved-dns-rr.h"
+#include "resolved-dns-synthesize.h"
+#include "resolved-manager.h"
+
+#include "log.h"
+#include "tests.h"
```

Comment:
these are all internal headers, so use the same block above to list them, sorted alphabetically